packages/flutter/lib/src/material/color_scheme.dart

```diff
@@ -283,9 +283,14 @@ class ColorScheme with Diagnosticable {
   /// If the resulting color scheme is too dark, consider setting `dynamicSchemeVariant`
   /// to [DynamicSchemeVariant.fidelity], whose palettes match the seed color.
   ///
+  /// The `contrastLevel` parameter indicates the contrast level between color
```

Comment:
suggest mentioning that medium and high contrast from the M3 guidelines correspond to 0.5 and 1.0 respectively.

Reply:
Thanks for helping review this! Will update!